doc/lua_api.txt

```diff
@@ -2634,6 +2633,19 @@ Some types may inherit styles from parent types.
 * tabheader
     * noclip - boolean, set to true to allow the element to exceed formspec bounds.
     * textcolor - color. Default white.
+* box
+    * noclip - boolean, set to true to allow the element to exceed formspec bounds.
+        * Defaults to false in formspec_version version 3 or higher
+    * **Note**: `colors`, `bordercolors`, and `borderwidths` can either be a single value
+      like `red`, an array of two like `red,yellow` (applies to the top/bottom and
+      right/left or top-left/bottom-right and top-right/bottom-left), or an array of four
+      like `red,yellow,green,blue` (applies to the top or top-left and rotates around
+      clockwise). This works the same as CSS borders.
```

Comment:
Suggestion:
```
    * **Note**: `colors`, `bordercolors`, and `borderwidths` accept multiple input types:
        * Single value (ex. `#FF0`): All borders
        * Two values (ex. `red,#FFAAFF`): Top/border and left/right border
        * Four values: Top border and rotates around clockwise
        * This is comparable to CSS borders.
```

Q: How do you specify whether it starts from the top, or does gradients starting from top-left?

Reply:
Gradients do start at the top-left.  It's in the current documentation.